js/main.js

```diff
@@ -21,6 +25,15 @@ function getRandomInt(max) {
   return Math.floor(Math.random() * max);
 }
 
+function submitSearch(event) {
+  event.preventDefault();
+  var mobileSearch = $mobileSearchContent.value;
+  var desktopSearch = $desktopSearchContent.value;
+  if (mobileSearch || desktopSearch !== '') {
```

Comment:
In a `submit` event handler, the `event.target` is the `form` element that was submitted. You should use `event.target.elements` to access the value of the input in the form that was submitted, so you don't need this conditional. Make sure you give each `input` element a `name` attribute in the HTML with the same value so you can use the same property name to access either one.